airbyte-integrations/connectors/destination-dev-null/README.md

```diff
@@ -1,6 +1,6 @@
-# Destination Dev Null
+# End-to-End Testing Destination
```

Comment:
bad merge? I think we want to keep the DevNull name, no?

Reply:
Yes, I haven't picked up your changes yet, I'm going to just take your readme.

Reply:
https://github.com/airbytehq/airbyte/pull/45704#discussion_r1769135880